pkgs/development/python-modules/pybind11/default.nix

```diff
@@ -34,9 +34,9 @@ buildPythonPackage rec {
     "-DBoost_INCLUDE_DIR=${lib.getDev boost}/include"
     "-DEIGEN3_INCLUDE_DIR=${lib.getDev eigen}/include/eigen3"
     "-DBUILD_TESTING=on"
-  ] ++ lib.optionals (python.isPy3k && !stdenv.cc.isClang) [
-    "-DPYBIND11_CXX_STANDARD=-std=c++17"
-  ];
+    "-DPYTHON_EXECUTABLE:FILEPATH=${python.pythonForBuild}/bin/python"
```

Comment:
```suggestion
    "-DPYTHON_EXECUTABLE:FILEPATH=${python.pythonForBuild.interpreter}"
```

Reply:
- Changed according to suggestion. tested again building aarch64 cross and x86_64 native
- rebased on staging
- will change target branch in PR now
Thanks! (and sorry, I was not aware the procedure with staging and that I trigger many rebuilds with this. makes total sense now).